samples/data_labelling.html

```diff
@@ -30,12 +30,12 @@
                 backgroundColor: color(window.chartColors.red).alpha(0.2).rgbString(),
                 borderColor: window.chartColors.red,
                 data: [
-                    randomScalingFactor(), 
```

Comment:
Yes, my editor is configured to remove trailing whitespace, which is actually an [ESLint rule,](/chartjs/Chart.js/blob/5c5b1bfe59438275330cc03a0fcb01104498535f/.eslintrc#L178) but examples are not part of the lint process.